x-pack/legacy/plugins/epm/server/lib/field.test.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { readFileSync, writeFileSync } from 'fs';
+import path from 'path';
+import { safeLoad } from 'js-yaml';
+import { Field, ProcessFields } from './field';
+
+test('tests loading fields.yml', () => {
+  const fieldsYML = readFileSync(path.join(__dirname, '/tests/fields/base.yml'), 'utf-8');
+
+  const data = safeLoad(fieldsYML.toString());
+  const fields = data as Field[];
```

Comment:
The preference is to do a "normal" [annotation](https://www.typescriptlang.org/docs/handbook/typescript-in-5-minutes.html#type-annotations) when you can instead of "casting" to a type with an [assertion](https://www.typescriptlang.org/docs/handbook/basic-types.html#type-assertions)

```typescript
const fields: Field[] = data;
```
over
```typescript
const fields = data as Field[];
```

However, since you don't use `data` anywhere else, when you combine it with the previous suggestion you can combine the two lines into

```typescript
const fields: Field[] = safeLoad(fieldsYML);
```